src/handler_sse.c

```diff
@@ -28,7 +28,6 @@ error_t handleApiSse(HttpConnection *connection, const char_t *uri, const char_t
         mutex_unlock(MUTEX_SSE_CTX);
         TRACE_ERROR("All slots full, in total %" PRIu8 " clients\r\n", sseSubscriptionCount);
         httpInitResponseHeader(connection);
-        connection->response.contentLength = 0;
```

Comment:
The content length is zero, as there is no body. Or did I miss something?

Reply:
I readded the content length for the 503 code

As I understand the function, it is providing a stream of events.
A stream has no explizit size, so we can omit the content length entirely

Why This Works:
* Chunked Transfer Encoding: This allows the server to send data in chunks without specifying the total length upfront. The client (or reverse proxy) knows to keep the connection open and process each chunk as it arrives.
* No Misleading Content-Length: By not setting Content-Length, you avoid confusing the client or reverse proxy about the nature of the response.

when i run this code my browser constantly receives `keep-alive	{ "type":"keep-alive", "data":"" }`

Currently I haven't connected a real box to the setup. Is this nessesary or is there a way to emulate a box?

Reply:
If there is a 503, there is no chunked encoding. To simulate a connection, you can just open the SSE endpoint. If you spam it, the error should occur if all slots are used.

A there are keep-alives, no need for a real box.

Reply:
in case of a 503 i changed the chunkedEncoding to false

is there something else need to be done?

Reply:
Spamming the enpoint works, got a 503